scripts/lib/common.sh

```diff
@@ -49,9 +49,11 @@ function sed_inplace() {
 
     if [[ "$OSTYPE" == "darwin"* ]]; then
         # macOS uses BSD sed
+        echo "sed -i '' \"s|$pattern|$replacement|g\" \"$file\""
```

Comment:
you might want to look into something like `set -x` instead

Reply:
ok, let me remove it, I think people are free to add `set -x` by themself when debugging